cmd/cephcsi.go

```diff
@@ -128,6 +128,9 @@ func init() {
 	flag.BoolVar(&conf.Version, "version", false, "Print cephcsi version information")
 	flag.BoolVar(&conf.EnableProfiling, "enableprofiling", false, "enable go profiling")
 
+	// CSI-Addons configuration
+	flag.StringVar(&conf.CSIAddonsEndpoint, "csi-addons-endpoint", "unix://tmp/csi-addons.sock", "CSI-Addons endpoint")
```

Comment:
question:- Do we need to start one more server? can't we serve the new Spec in the existing server?

Reply:
We could, but that does not look very clean to me. It are two distinct projects (csi, csi-addons), and maybe at one point (parts of) the csi-addons get included in the CSI spec. Keeping the separate prevents confusion and maybe even potential conflicts in the future.